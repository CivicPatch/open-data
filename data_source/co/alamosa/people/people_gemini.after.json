[
  {
    "name": "Jackie Vigil",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 3"
    ],
    "image": null,
    "phone_number": "(719) 992-8008",
    "email": null,
    "website": "https://cityofalamosa.org/mayorprotem-ward3",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/mayorprotem-ward3",
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/mayor"
    ],
    "updated_at": "2025-06-30T17:01:56Z"
  },
  {
    "name": "Ty Coleman",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "(719) 313-7241",
    "email": null,
    "website": "https://cityofalamosa.org/mayor",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/mayor"
    ],
    "updated_at": "2025-06-30T17:01:56Z"
  },
  {
    "name": "Michael Carson",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "At-large"
    ],
    "image": null,
    "phone_number": "(719) 992-4839",
    "email": null,
    "website": "https://cityofalamosa.org/councilor-at-large-daniel",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/councilor-at-large-daniel"
    ],
    "updated_at": "2025-06-30T17:01:56Z"
  },
  {
    "name": "Jan Vigil",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "At-large"
    ],
    "image": null,
    "phone_number": "(970) 946-5611",
    "email": null,
    "website": "https://cityofalamosa.org/councilor-at-large-vigil",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/councilor-at-large-vigil"
    ],
    "updated_at": "2025-06-30T17:01:56Z"
  },
  {
    "name": "Liz Hensley",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 1"
    ],
    "image": null,
    "phone_number": "(719) 580-0201",
    "email": null,
    "website": "https://cityofalamosa.org/city-councilor-ward-1",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/city-councilor-ward-1"
    ],
    "updated_at": "2025-06-30T17:01:56Z"
  },
  {
    "name": "Dawn Krebs",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 2"
    ],
    "image": null,
    "phone_number": "(719) 580-2194",
    "email": null,
    "website": "https://cityofalamosa.org/city-councilor-ward-2",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/city-councilor-ward-2"
    ],
    "updated_at": "2025-06-30T17:01:56Z"
  },
  {
    "name": "Jamie Dominguez",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 4"
    ],
    "image": null,
    "phone_number": "(719) 937-3039",
    "email": null,
    "website": "https://cityofalamosa.org/city-councilor-ward-4",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/city-councilor-ward-2",
      "https://cityofalamosa.org/city-councilor-ward-4"
    ],
    "updated_at": "2025-06-30T17:01:56Z"
  }
]