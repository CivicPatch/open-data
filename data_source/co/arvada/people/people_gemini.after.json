[
  {
    "name": "Brad Rupert",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "At-large"
    ],
    "image": null,
    "phone_number": null,
    "email": "Council@arvada.org",
    "website": null,
    "start_date": "2024-12-16",
    "end_date": "2025-11",
    "sources": [
      "https://www.arvada.org/1174/Brad-Rupert-Appointed-to-Fill-Vacant-At-",
      "https://www.arvada.org/158/About-City-Council"
    ],
    "updated_at": "2025-06-27T20:13:13Z"
  },
  {
    "name": "Lauren Simpson",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": "Council@arvada.org",
    "website": null,
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.arvada.org/1174/Brad-Rupert-Appointed-to-Fill-Vacant-At-",
      "https://www.arvada.org/158/About-City-Council"
    ],
    "updated_at": "2025-06-27T20:13:13Z"
  },
  {
    "name": "John Marriott",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "District 3"
    ],
    "image": null,
    "phone_number": null,
    "email": "Council@arvada.org",
    "website": null,
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.arvada.org/158/About-City-Council"
    ],
    "updated_at": "2025-06-27T20:13:13Z"
  },
  {
    "name": "Bob Fifer",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "District 4"
    ],
    "image": null,
    "phone_number": null,
    "email": "Council@arvada.org",
    "website": null,
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.arvada.org/158/About-City-Council"
    ],
    "updated_at": "2025-06-27T20:13:13Z"
  },
  {
    "name": "Randy Moorman",
    "roles": [
      "Mayor Pro Tempore"
    ],
    "divisions": [
      "District 1"
    ],
    "image": null,
    "phone_number": null,
    "email": "Council@arvada.org",
    "website": null,
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.arvada.org/158/About-City-Council"
    ],
    "updated_at": "2025-06-27T20:13:13Z"
  },
  {
    "name": "Sharon Davis",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "At-large"
    ],
    "image": null,
    "phone_number": null,
    "email": "Council@arvada.org",
    "website": null,
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.arvada.org/158/About-City-Council"
    ],
    "updated_at": "2025-06-27T20:13:13Z"
  },
  {
    "name": "Shawna Ambrose",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "District 2"
    ],
    "image": null,
    "phone_number": null,
    "email": "Council@arvada.org",
    "website": null,
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.arvada.org/158/About-City-Council"
    ],
    "updated_at": "2025-06-27T20:13:13Z"
  }
]