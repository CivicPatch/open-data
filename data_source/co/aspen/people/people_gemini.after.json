[
  {
    "name": "Rachael E. Richards",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "(970) 471-8329",
    "email": null,
    "website": null,
    "start_date": "2025-04",
    "end_date": "2027-04",
    "sources": [
      "https://aspen.gov/674/City-Council"
    ],
    "updated_at": "2025-06-30T18:41:56Z"
  },
  {
    "name": "John Doyle",
    "roles": [
      "Mayor Pro Tempore"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "(970) 948-6251",
    "email": null,
    "website": null,
    "start_date": "2025-04",
    "end_date": "2029-04",
    "sources": [
      "https://aspen.gov/674/City-Council"
    ],
    "updated_at": "2025-06-30T18:41:56Z"
  },
  {
    "name": "Bill Guth",
    "roles": [
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "(970) 300-2120",
    "email": null,
    "website": null,
    "start_date": "2023-04",
    "end_date": "2027-04",
    "sources": [
      "https://aspen.gov/674/City-Council"
    ],
    "updated_at": "2025-06-30T18:41:56Z"
  },
  {
    "name": "Sam Rose",
    "roles": [
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "(802) 752-7026",
    "email": null,
    "website": null,
    "start_date": "2023-04",
    "end_date": "2027-04",
    "sources": [
      "https://aspen.gov/674/City-Council"
    ],
    "updated_at": "2025-06-30T18:41:56Z"
  },
  {
    "name": "Christine Benedetti",
    "roles": [
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "(206) 349-5212",
    "email": null,
    "website": null,
    "start_date": "2025-04",
    "end_date": "2029-04",
    "sources": [
      "https://aspen.gov/674/City-Council"
    ],
    "updated_at": "2025-06-30T18:41:56Z"
  }
]