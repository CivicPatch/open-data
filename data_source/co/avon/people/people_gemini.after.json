[
  {
    "name": "Tamra Underwood",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": "tunderwood@avon.org",
    "website": "https://www.avon.org/directory.aspx?EID=163",
    "start_date": "2022",
    "end_date": "2026",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council"
    ],
    "updated_at": "2025-06-30T17:33:50Z"
  },
  {
    "name": "Rich Carroll",
    "roles": [
      "Mayor Pro Tempore"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": "rcarroll@avon.org",
    "website": "https://www.avon.org/directory.aspx?EID=7",
    "start_date": "2022",
    "end_date": "2026",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council",
      "https://www.avon.org/directory.aspx?EID=7"
    ],
    "updated_at": "2025-06-30T17:33:50Z"
  },
  {
    "name": "Chico Thuon",
    "roles": [
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": "cthuon@avon.org",
    "website": null,
    "start_date": "2022",
    "end_date": "2026",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council"
    ],
    "updated_at": "2025-06-30T17:33:50Z"
  },
  {
    "name": "Lindsay Hardy",
    "roles": [
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": "lhardy@avon.org",
    "website": "https://www.avon.org/directory.aspx?EID=204",
    "start_date": "2024",
    "end_date": "2028",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council",
      "https://www.avon.org/directory.aspx?EID=204"
    ],
    "updated_at": "2025-06-30T17:33:50Z"
  },
  {
    "name": "Ruth Stanley",
    "roles": [
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": "rstanley@avon.org",
    "website": "https://www.avon.org/directory.aspx?EID=339",
    "start_date": "2022",
    "end_date": "2026",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council",
      "https://www.avon.org/directory.aspx?EID=339"
    ],
    "updated_at": "2025-06-30T17:33:50Z"
  },
  {
    "name": "Gary Brooks",
    "roles": [
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": "gbrooks@avon.org",
    "website": "https://www.avon.org/directory.aspx?EID=373",
    "start_date": "2024",
    "end_date": "2028",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council",
      "https://www.avon.org/directory.aspx?EID=373"
    ],
    "updated_at": "2025-06-30T17:33:50Z"
  },
  {
    "name": "Kevin Hyatt",
    "roles": [
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": "khyatt@avon.org",
    "website": null,
    "start_date": "2024",
    "end_date": "2028",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council"
    ],
    "updated_at": "2025-06-30T17:33:50Z"
  }
]