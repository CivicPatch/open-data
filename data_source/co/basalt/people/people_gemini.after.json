[
  {
    "name": "David Knight",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "(970) 924-0475",
    "email": "David.Knight@basalt.net",
    "website": "https://www.basalt.net/Directory.aspx?EID=111",
    "start_date": "2024-04",
    "end_date": "2028-04",
    "sources": [
      "https://www.basalt.net/137/Town-Council",
      "https://www.basalt.net/Directory.aspx?EID=111"
    ],
    "updated_at": "2025-06-30T21:42:08Z"
  },
  {
    "name": "Ryan Slack",
    "roles": [
      "Mayor Pro Tempore"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "(970) 927-4701",
    "email": "ryan.slack@basalt.net",
    "website": "https://www.basalt.net/directory.aspx?EID=96",
    "start_date": "2022-04",
    "end_date": "2026-04",
    "sources": [
      "https://www.basalt.net/137/Town-Council",
      "https://www.basalt.net/directory.aspx?EID=96"
    ],
    "updated_at": "2025-06-30T21:42:08Z"
  },
  {
    "name": "Angela Anderson",
    "roles": [
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "(970) 279-3310",
    "email": "Angela.Anderson@basalt.net",
    "website": "https://www.basalt.net/directory.aspx?EID=122",
    "start_date": "2022-04",
    "end_date": "2026-04",
    "sources": [
      "https://www.basalt.net/137/Town-Council",
      "https://www.basalt.net/directory.aspx?EID=122",
      "https://www.basalt.net/Directory.aspx?DID=11"
    ],
    "updated_at": "2025-06-30T21:42:08Z"
  },
  {
    "name": "Dieter Schindler",
    "roles": [
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "(773) 578-7702",
    "email": null,
    "website": "https://www.basalt.net/Directory.aspx?EID=123",
    "start_date": "2022-04",
    "end_date": null,
    "sources": [
      "https://www.basalt.net/137/Town-Council",
      "https://www.basalt.net/Directory.aspx?EID=123"
    ],
    "updated_at": "2025-06-30T21:42:08Z"
  },
  {
    "name": "Rick Stevens",
    "roles": [
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "(970) 379-2752",
    "email": null,
    "website": "https://www.basalt.net/Directory.aspx?EID=145",
    "start_date": "2024-04",
    "end_date": null,
    "sources": [
      "https://www.basalt.net/137/Town-Council",
      "https://www.basalt.net/Directory.aspx?EID=145"
    ],
    "updated_at": "2025-06-30T21:42:08Z"
  },
  {
    "name": "Hannah Berman",
    "roles": [
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "(303) 727-0366",
    "email": "Hannah.Berman@basalt.net",
    "website": "https://www.basalt.net/Directory.aspx?EID=143",
    "start_date": "2024-04",
    "end_date": "2028-04",
    "sources": [
      "https://www.basalt.net/137/Town-Council",
      "https://www.basalt.net/Directory.aspx?EID=143"
    ],
    "updated_at": "2025-06-30T21:42:08Z"
  },
  {
    "name": "Angele Dupre-Butchart",
    "roles": [
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "(713) 410-2373",
    "email": "Angele.Dupre-Butchart@basalt.net",
    "website": "https://www.basalt.net/Directory.aspx?EID=144",
    "start_date": "2024-04",
    "end_date": "2028-04",
    "sources": [
      "https://www.basalt.net/137/Town-Council",
      "https://www.basalt.net/Directory.aspx?EID=144"
    ],
    "updated_at": "2025-06-30T21:42:08Z"
  }
]