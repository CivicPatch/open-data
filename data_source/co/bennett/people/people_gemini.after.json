[
  {
    "name": "Whitney Oakley",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.bennettco.gov/town-board-trustees/directory-listing/whitney-oakley",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.bennettco.gov/town-board-trustees",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/whitney-oakley",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/kevin-barden",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/steve-dambroski",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/royce-pindell",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/denice-smith",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/donna-sus",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/larry-vittum"
    ],
    "updated_at": "2025-06-30T22:15:22Z"
  },
  {
    "name": "Kevin Barden",
    "roles": [
      "Mayor Pro Tempore",
      "Trustee"
    ],
    "divisions": [
      "At-large"
    ],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.bennettco.gov/town-board-trustees/directory-listing/kevin-barden",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.bennettco.gov/town-board-trustees",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/whitney-oakley",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/kevin-barden",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/steve-dambroski",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/royce-pindell",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/denice-smith",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/donna-sus",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/larry-vittum"
    ],
    "updated_at": "2025-06-30T22:15:22Z"
  },
  {
    "name": "Steve Dambroski",
    "roles": [
      "Trustee"
    ],
    "divisions": [
      "At-large"
    ],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.bennettco.gov/town-board-trustees/directory-listing/steve-dambroski",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.bennettco.gov/town-board-trustees",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/whitney-oakley",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/kevin-barden",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/steve-dambroski",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/royce-pindell",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/denice-smith",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/donna-sus",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/larry-vittum"
    ],
    "updated_at": "2025-06-30T22:15:22Z"
  },
  {
    "name": "Royce Pindell",
    "roles": [
      "Trustee"
    ],
    "divisions": [
      "At-large"
    ],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.bennettco.gov/town-board-trustees/directory-listing/royce-pindell",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.bennettco.gov/town-board-trustees",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/whitney-oakley",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/kevin-barden",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/steve-dambroski",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/royce-pindell",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/denice-smith",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/donna-sus",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/larry-vittum"
    ],
    "updated_at": "2025-06-30T22:15:22Z"
  },
  {
    "name": "Denice Smith",
    "roles": [
      "Trustee"
    ],
    "divisions": [
      "At-large"
    ],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.bennettco.gov/town-board-trustees/directory-listing/denice-smith",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.bennettco.gov/town-board-trustees",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/whitney-oakley",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/kevin-barden",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/steve-dambroski",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/royce-pindell",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/denice-smith",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/donna-sus",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/larry-vittum"
    ],
    "updated_at": "2025-06-30T22:15:22Z"
  },
  {
    "name": "Donna Sus",
    "roles": [
      "Trustee"
    ],
    "divisions": [
      "At-large"
    ],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.bennettco.gov/town-board-trustees/directory-listing/donna-sus",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.bennettco.gov/town-board-trustees",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/whitney-oakley",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/kevin-barden",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/steve-dambroski",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/royce-pindell",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/denice-smith",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/donna-sus",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/larry-vittum"
    ],
    "updated_at": "2025-06-30T22:15:22Z"
  },
  {
    "name": "Larry Vittum",
    "roles": [
      "Trustee"
    ],
    "divisions": [
      "At-large"
    ],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.bennettco.gov/town-board-trustees/directory-listing/larry-vittum",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.bennettco.gov/town-board-trustees",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/whitney-oakley",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/kevin-barden",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/steve-dambroski",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/royce-pindell",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/denice-smith",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/donna-sus",
      "https://www.bennettco.gov/town-board-trustees/directory-listing/larry-vittum"
    ],
    "updated_at": "2025-06-30T22:15:22Z"
  }
]