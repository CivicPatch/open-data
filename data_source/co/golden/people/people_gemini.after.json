[
  {
    "name": "Bill Fisher",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 4"
    ],
    "image": null,
    "phone_number": null,
    "email": "BFisher@cityofgolden.net",
    "website": "https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net",
      "https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php"
    ],
    "updated_at": "2025-06-28T18:44:23Z"
  },
  {
    "name": "Laura Weinberg",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:44:23Z"
  },
  {
    "name": "Lisa Vitry",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "District 1"
    ],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:44:23Z"
  },
  {
    "name": "Patty Evans",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "District 2"
    ],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:44:23Z"
  },
  {
    "name": "Robert Reed",
    "roles": [
      "Council Member",
      "Mayor Pro Tempore"
    ],
    "divisions": [
      "Ward 1"
    ],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:44:23Z"
  },
  {
    "name": "Paul Haseman",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 2"
    ],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:44:23Z"
  },
  {
    "name": "Don Cameron",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 3"
    ],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:44:23Z"
  }
]